packages/wasm-sdk/src/context_provider.rs

```diff
@@ -1,18 +1,27 @@
 use std::sync::Arc;
 
+use dash_sdk::platform::ContextProvider;
 use dash_sdk::{
     dpp::{
+        data_contract::TokenConfiguration,
         prelude::CoreBlockHeight,
         util::vec::{decode_hex, encode_hex},
+        version::PlatformVersion,
     },
     error::ContextProviderError,
     platform::{DataContract, Identifier},
 };
-use drive_proof_verifier::ContextProvider;
 use wasm_bindgen::prelude::wasm_bindgen;
 
 #[wasm_bindgen]
+#[derive(Clone)]
 pub struct WasmContext {}
+
+/// A wrapper for TrustedHttpContextProvider that works in WASM
+#[derive(Clone)]
+pub struct WasmTrustedContext {
```

Comment:
Why do we need both WasmContext and WasmTrustedContext?

I guess we can get rid of one of them, and also remove that QUORUM_KEYS constant and related logic.